docker/Dockerfile

```diff
@@ -23,10 +23,11 @@ ARG celeborn_gid=10006
 
 ENV CELEBORN_HOME=/opt/celeborn
 ENV PATH=/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/busybox:${CELEBORN_HOME}/sbin:${CELEBORN_HOME}/bin
+ENV LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so
 
 RUN set -ex && \
     apt-get update && \
-    apt-get install -y bash tini busybox bind9-utils telnet net-tools procps krb5-user dnsutils && \
+    apt-get install -y bash tini busybox bind9-utils telnet net-tools procps krb5-user dnsutils gpg libsnappy1v5 gettext-base libjemalloc-dev && \
```

Comment:
why `gpg` `libsnappy1v5` `gettext-base` are required?

```
➜  ~ apt depends libjemalloc-dev
libjemalloc-dev
  Depends: libjemalloc2 (= 5.2.1-1ubuntu1)
  Suggests: binutils
➜  ~ apt depends libjemalloc2
libjemalloc2
  Depends: libc6 (>= 2.29)
  Depends: libgcc-s1 (>= 3.3)
  Depends: libstdc++6 (>= 4.4.0)
```

Reply:
@pan3793 Sorry for this, there's no need for introducing these three libs, I've fixed it.

Reply:
seems the runtime library `libjemalloc2` is enough, as we don't need dev stuff

Reply:
`libjemalloc-dev` could help [profiling the memory](https://github.com/jemalloc/jemalloc/wiki/Use-Case%3A-Heap-Profiling) to analysis the native memory leak problem. This is extremely helpful for complex OOMKilled problem.

Reply:
Thanks for information, let's keep `libjemalloc-dev` then